docs/UserGuide.adoc

```diff
@@ -373,18 +379,21 @@ their tags. As you can search using incomplete keywords, the intended search for
 Please note that the search is case-insensitive, an example being the `Sweet` tag of `Bandung`. Moreover,
 as with Example 2, we allow for incomplete words to be used as search keywords.
 
+[[goal]]
 === `goal` : Setting a daily calorie target
 (by Vineeth)
 
 A healthy lifestyle is not achieved overnight. Calgo understands the importance of keeping the end in mind with regards to achieving your healthy lifestyle.
-Thus, to motivate you to put in consistent effort, you can set daily caloric goals for yourself using the `goal` command.
+Thus, to motivate you to put in consistent effort, you can set daily calorie goals for yourself using the `goal` command.
 
 Here are some key pointers:
 
 ****
-* After setting a daily calorie goal, you can still update it subsequently!
+* The goal that you set will be used to generate insights in your consumption report. For more information related to that, you check out the `report` command <<report, here>>.
```

Comment:
In general, very nicely done Vineeth! The standardisation the team discussed yesterday, and most of the tips we all agreed on for better readability and consistency are all applied!